### PR TITLE
Allow build tags

### LINKED ISSUE #21: Build tags support.
This is a feature request.

It'd be nice to add a flag to be able to specify custom build tags.

For example, I have some code behind a `// +build dev` constraint that I'd like to run `unconvert` on.

### PR DESCRIPTION
**Pull request by yoheimuta**
closes #21

### CODE REVIEW

File: unconvert.go [hunk 1]
```diff
@@ -292,12 +294,16 @@ func (noImporter) Import(path string) (*types.Package, error) {
...
 }
 
-func computeEdits(importPaths []string, os, arch string, cgoEnabled bool) fileToEditSet {
+func computeEdits(importPaths []string, os, arch string, cgoEnabled bool, buildTags string) fileToEditSet {
 	ctxt := build.Default
 	ctxt.GOOS = os
 	ctxt.GOARCH = arch
 	ctxt.CgoEnabled = cgoEnabled
 
+	if 0 < len(buildTags) {
```
Comment (shurcooL) [inline unconvert.go]: FWIW, I [personally think](https://dmitri.shuralyov.com/idiomatic-go#empty-string-check) it's more readable (and common) to write this as `if buildTags != "" {`.

File: unconvert.go [hunk 2]
```diff
@@ -166,8 +167,9 @@ var (
 	flagApply      = flag.Bool("apply", false, "apply edits to source files")
 	flagCPUProfile = flag.String("cpuprofile", "", "write CPU profile to file")
 	// TODO(mdempsky): Better description and maybe flag name.
-	flagSafe = flag.Bool("safe", false, "be more conservative (experimental)")
-	flagV    = flag.Bool("v", false, "verbose output")
+	flagSafe      = flag.Bool("safe", false, "be more conservative (experimental)")
+	flagV         = flag.Bool("v", false, "verbose output")
+	flagBuildTags = flag.String("tags", "", "a space-separated list of build tags. see also 'go build' option for details")
```
Comment (shurcooL) [inline unconvert.go]: You might want to consider using [`buildutil.TagsFlag`](https://godoc.org/golang.org/x/tools/go/buildutil#TagsFlag) here.

File: unconvert.go (hunk 1 above)
Comment (shurcooL) [reply unconvert.go]: Additionally, the `if` statement can be completely removed without changing behavior.

If `buildTags` is the empty string, then `strings.Fields` returns an empty slice, which is what `ctxt.BuildTags` is anyway. So the `if` statement is not neccessary.

**Comment by yoheimuta** [comment]
Thank you for a review.
I did not know `buildutil.TagsFlag`, so I fixed it using `buildutil.TagsFlag`.

**Comment by mvdan** [comment]
Ping @mdempsky?

**Comment by shurcooL** [comment]
Friendly ping @mdempsky. I ran into this need today, but got:

```
$ unconvert -tags=dev ./...
flag provided but not defined: -tags
usage: unconvert [flags] [package ...]
  -all
    	type check all GOOS and GOARCH combinations
  -apply
    	apply edits to source files
  -cpuprofile string
    	write CPU profile to file
  -safe
    	be more conservative (experimental)
  -v	verbose output
```

**Pull request merged**